content/gage_height.md

```diff
@@ -171,19 +158,15 @@ the [Flood Inundation Mapper](https://fim.wim.usgs.gov/fim/) tool or the
 tool to visualize what flooding will look like at different gage
 heights.
 
-## 
-
-
 <div class="grid-row">
 
-{{< figure src="/static/gage_height/5.png" caption="A hydrograph of the [Tangipahoa River at Robert,LA](https://waterdata.usgs.gov/monitoring-location/07375500/#parameterCode=00065&startDT=2021-06-06&endDT=2021-06-10) with Flood Stages visible." alt="A graph showing gage height over time and a varying river
-level. Horizontal lines on the graph indicate the National Weather Service flood levels." >}}
+{{< figure src="/static/gage_height/5.png" 
+caption=`A hydrograph of the [Tangipahoa River at Robert,LA](https://waterdata.usgs.gov/monitoring-location/07375500/#parameterCode=00065&startDT=2021-06-06&endDT=2021-06-10) with Flood Stages visible.` 
```

Comment:
You can use string literals to have multi line attributes (start and end string with backticks)